File: 9-palindrome-number/9-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
  if(x < 0 || (x !== 0 && x % 10 == 0))
        return false
  const str = ''+x
  return str === [...str].reverse().join('')
};

File: 9-palindrome-number/9-palindrome-number.cs
public class Solution {
  public bool IsPalindrome(int x) {
    string str = x.ToString();
    int i = 0;
    int j = str.Length - 1;
    for (;i < j; i++, j--) {
      if (str[i] != str[j]) return false;
    }
    return true;
  }
}